frontend/src/types/question.d.ts

```diff
@@ -1,6 +1,7 @@
 export type Example = {
   input: string;
   output: string;
+  explanation: string;
```

Comment:
same idea, if we want to split the explanation here, we might also need to consider a field for images, and I'm not sure if there's standardization for these details in leetcode or not. need further discussion.

Reply:
Oh yea you are right, ideally we should have an image field for example so that we can load the tailwindcss classes correctly. However, let's skip this for now as I have no idea about what information we can retrieve from LeetCode too.